src/bootstrap/config.rs

```diff
@@ -529,6 +530,7 @@ impl Config {
             set(&mut config.rust_dist_src, rust.dist_src);
             set(&mut config.quiet_tests, rust.quiet_tests);
             set(&mut config.test_miri, rust.test_miri);
+            set(&mut config.incremental, rust.incremental);
```

Comment:
This isn't quite right, since this means that the configuration, if present, will override the command line flags. Instead we should have the command line flags override the configuration.

(I can provide instructions how best to do this if you need help, just let me know).

Reply:
I am aware of that, but there's no flag for disabling incremental. Either the command is there, or it's not. Or are you talking about the case where the config explicitly says "false" but -i is passed?